canonical-websites/www.ubuntu.com
# Cloud economics campaign landing page

## Done

- New cloud economics campaign landing page at `/engage/cloud-economics`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/engage/cloud-economics](http://0.0.0.0:8001/engage/cloud-economics)
- Ensure the content matches the [copy doc](https://docs.google.com/document/d/1NtRmAB4Pkphc8WTU8n4-b10knVqqFhMDc_FtGAwQ7nw/edit?usp=sharing)

## Screenshots

![](https://screenshotscdn.firefoxusercontent.com/images/1575af04-f8a2-4f74-96d8-2f05d82794cf.jpg)
